ee/clickhouse/queries/test/test_trends.py

```diff
@@ -343,6 +343,40 @@ def test_dau_with_breakdown_filtering_with_prop_filter(self):
 
         self.assertEntityResponseEqual(action_response, event_response)
 
+    @test_with_materialized_columns(event_properties=["$host"], person_properties=["$some_prop"])
+    def test_against_clashing_entity_and_property_filter_naming(self):
```

Comment:
What is the clash here? I was thinking that maybe `$some_prop` would be an event property and a person property but it doesn't look like that from the test.

Reply:
Good question. 

So ee/clickhouse/queries/breakdown_props.py used to have this code:

```python

def get_breakdown_person_prop_values(
    filter: Filter, entity: Entity, aggregate_operation: str, team_id: int, limit: int = 25, extra_params={}
):
    _event_table_name = "e"
    parsed_date_from, parsed_date_to, _ = parse_timestamps(filter=filter, team_id=team_id)
    prop_filters, prop_filter_params = parse_prop_clauses(
        filter.properties,
        team_id,
        table_name=_event_table_name,
        filter_test_accounts=filter.filter_test_accounts,
        person_properties_column="person_props",
        allow_denormalized_props=True,
    )

    entity_params, entity_format_params = get_entity_filtering_params(
        entity, team_id, table_name=_event_table_name, with_prop_filters=True, person_properties_column="person_props"
    )
```

This results in two calls to `parse_prop_clauses` - one direct, one via `get_entity_filtering_params`. Since prepend was the same for both they both generated the same params key `k_global_0` or smth - which blew up.

In the test the same would happen - the two filters would result in same param name and crash due to type error.

My other breakdowns refactor also incidentally fixed this, but this PR adds an explicit test for it.